articles/container-service/container-service-connect.md

```diff
@@ -71,53 +71,64 @@ For further instructions you can see the [Kubernetes quick start](http://kuberne
 
 ## Connecting to a DC/OS or Swarm cluster
 
-The DC/OS and Docker Swarm clusters that are deployed by Azure Container Service expose REST endpoints. However, these endpoints are not open to the outside world. In order to manage these endpoints, you must create a Secure Shell (SSH) tunnel. After an SSH tunnel has been established, you can run commands against the cluster endpoints and view the cluster UI through a browser on your own system. This document walks you through creating an SSH tunnel from Linux, OS X, and Windows.
+The DC/OS and Docker Swarm clusters deployed by Azure Container Service provide several HTTP endpoints locally, but these are not accessible by default from the Internet. In order to use them, you must first create a Secure Shell (SSH) tunnel to an internal system. After this tunnel has been established, you can run commands which use these HTTP endpoints and view the cluster's Web interface from your local system.
+
+This document walks you through creating an SSH tunnel from Linux, OS X, and Windows clients.
 
 > [!NOTE]
-> You can create an SSH session with a cluster management system. However, we don't recommend this. Working directly on a management system exposes the risk for inadvertent configuration changes.   
+> These instructions focus on tunnelling TCP traffic over SSH. You can also start an interactive SSH session with one of the internal cluster management systems, but we don't recommend this. Working directly on an internal system risks inadvertent configuration changes.   
 > 
 > 
 
 ## Create an SSH tunnel on Linux or OS X
-The first thing that you do when you create an SSH tunnel on Linux or OS X is to locate the public DNS name of load-balanced masters. To do this, expand the resource group so that each resource is being displayed. Locate and select the public IP address of the master. This will open up a blade that contains information about the public IP address, which includes the DNS name. Save this name for later use. <br />
 
-![Public DNS name](media/pubdns.png)
+1. Get the public DNS name of the load balancer for the master group.
 
-Now open a shell and run the following command where:
+  * To do this in the portal, expand the resource group so that each resource is being displayed. Locate and select the public IP address of the master. This will open up a blade that contains information about the public IP address, which includes the DNS name. Save this name for later use. <br />
```

Comment:
The master DNS name is now visible on the container service overview blade. So this instruction should really be "Open the container service blade and view the Master FQDN in the overview section" or similar